server/src/handler/mod.rs

```diff
@@ -6,11 +6,17 @@ use tpm2_rs_base::errors::TpmRcError;
 
 /// The context that all command handler functions are given access to in order for them to process
 /// their given command.
-pub struct CommandHandler<'a, Deps: TpmContextDeps> {
+pub struct CommandHandler<Deps: TpmContextDeps> {
     /// Gives access to cryptographic operations.
-    pub crypto: &'a mut Deps::Crypto,
+    pub crypto: Deps::Crypto,
```

Comment:
Optional: Since this is `pub` anyways, part of me thinks we just shouldn't have a `new` function and should instead construct using names for clarity.